Documented purposes of mailing list links

adamchainz opened the pull request:
I noticed that there was no solid documentation on how to use the 'subscription email address'. Google seem to have hidden/taken down documentation on how these work too (the best source I found was this SO answer: https://webapps.stackexchange.com/a/15593 ) so I'm adding an explicit how-to, plus clarifying the purposes of the other links.

timgraham commented:
I wonder if we should switch to something like a table so that the instructions don't have to be repeated each time. I'm especially thinking of documentation translators.

tobiaswiese commented:
Are you sure this works. A week ago or so I tried to use change the email address I use for django-developers@ to use my normal email address instead of Gmail it let me subscribe, but when I tried to post something it didn't let me.

adamchainz commented:
@tobiaswiese it sounds like if you could subscribe it's working. Sure your emails aren't being rerouted through another address or similar weirdness?

tobiaswiese commented:
Yes I already got mail delivered to that address, and I'm in full control of the handling mail server. Just sending from that address wasn't possible.

I also later added the address to my google account, which made it possible to select the address in Google Groups web interface, and now Sending and Receiving is possible.

adamchainz commented:
That's disappointing. I found [the support page](https://support.google.com/groups/answer/1067205?hl=en) that lists:

> Join a Google group without a Gmail address
> If you don't have a Google Account, you can:
> 
> * Read posts in public groups
> * Search for posts in public groups
>
> You need a Google Account to:
> 
> * Create and manage a group
> * Join a group
> * Post to a group
> * Delete a post
> * Read a restricted group's posts

I'll update the PR to point to this page and point out these caveats.

adamchainz commented:
I've added a link to the Google Groups documentation on joining and updated and rearranged the link descriptions.

I tried a table but there's too much information to fit in comfortably. I reduced the amount of text in the link descriptions and since it's repeated, it shouldn't be too much of a translation burden.e

adamchainz commented:
Yeah, I agree this is less needed now, especially since more queries are ending up in the forum.
